packages/api-specification/api/application.ts

```diff
@@ -0,0 +1,33 @@
+export interface ApplicationAPI {

```

Comment:
I guess a wider question is, should the API support multiple applications?

With OpenFin you can create a [new Application instance](http://cdn.openfin.co/jsdocs/stable/fin.desktop.Application.html#Application) through the JavaScript API.

Personally I think this is a bit of an edge case. I can't see when you'd want to do this in practice? I'd be happy with ContainerJS treating the Application as a singleton.